opm/output/data/Aquifer.hpp

```diff
@@ -34,81 +32,128 @@ namespace Opm { namespace data {
      */
     enum class AquiferType
     {
-        Fetkovich, CarterTracey, Numerical,
+        Fetkovich, CarterTracy, Numerical,
     };
 
-    struct FetkovichData {
+    struct FetkovichData
+    {
         double initVolume;
         double prodIndex;
         double timeConstant;
     };
 
-    struct AquiferData {
-        int aquiferID = 0;    //< One-based ID, range 1..NANAQ
-        double pressure = 0.0;  //< Aquifer pressure
-        double fluxRate = 0.0; //< Aquifer influx rate (liquid aquifer)
-        // TODO: volume should have a better name, since meaning not clear
-        double volume = 0.0;    //< Produced liquid volume
-        double initPressure = 0.0;    //< Aquifer's initial pressure
-        double datumDepth = 0.0;      //< Aquifer's pressure reference depth
+    struct CarterTracyData
+    {
+        double dimensionless_time;
+        double dimensionless_pressure;
+    };
+
+    struct AquiferData
+    {
+        int aquiferID = 0;         //< One-based ID, range 1..NANAQ
+        double pressure = 0.0;     //< Aquifer pressure
+        double fluxRate = 0.0;     //< Aquifer influx rate (liquid aquifer)
+        double volume = 0.0;       //< Produced liquid volume
+        double initPressure = 0.0; //< Aquifer's initial pressure
+        double datumDepth = 0.0;   //< Aquifer's pressure reference depth
 
         AquiferType type;
         std::shared_ptr<FetkovichData> aquFet;
+        std::shared_ptr<CarterTracyData> aquCT;
```

Comment:
I have a feeling `std::optional<>` is more "correct" than `std::shared_ptr<>`, or maybe `std::variant<>`?

Reply:
> have a feeling `std::optional<>` is more "correct" than `std::shared_ptr<>`, or maybe `std::variant<>`?

Yes, probably, but that's a bigger refactoring which affects the restart file reader too.  I'd rather do that as a separate PR later.